jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/assignment/LoadCalculator.java

```diff
@@ -0,0 +1,13 @@
+package org.jbpm.services.task.assignment;
+
+import java.util.Collection;
+import java.util.List;
+
+import org.kie.api.task.TaskContext;
+import org.kie.api.task.model.User;
+
+public interface LoadCalculator {
```

Comment:
we can leave it for now, please just add license header as it seems to be missing here